docs/contributing/handling-deleted-urls.md

```diff
@@ -0,0 +1,63 @@
+# Handling deleted URLs
+
+When removing or renaming a page or folder previously associated with a URL route on the website, you need to either redirect it or replace it with an 'archived' page.
+
+For example, renaming `community/juggling.md` to `community/circus-performance.md` means that the URL **design-system.service.gov.uk/community/juggling/** no longer exists. You need to provide a way for user's to reach the new URL or section, or explain why the old URL no longer exists.
+
+This is important because you should not leave URLs that:
+
+- were previously public
+- might be linked elsewhere on the internet
+- are unhandled and will disrupt a user's journey
+
+For example, search engines take time to reindex pages that no longer exist and the removed pages might continue to appear in search results for some time after you remove them.
+
+## Choosing between redirecting or archiving
+
+There are 2 methods we can use to direct users to a new URL:
+
+- Redirecting
+- Archiving
+
+You should use redirecting when you are just renaming something and/or the content of a given page or section is the same.
+
+You should use archiving when you have consciously removed a page or section or the content structure of a section has changed. You can still redirect a user to a new page or section or provide an alternative when you archive something but you may need to explain why.
+
+## Creating a redirect
+
+301 redirects can be created via [netlify's redirect config options](https://docs.netlify.com/routing/redirects/). To do this:
+
+1. Go to our [netlify.toml file](../../netlify.toml)
+2. Under the comment `# Redirect pages that have moved` add the following:
+   ```plaintext
```

Comment:
Can we add an example of this? My interpretation is that we're just adding the path, but good to make it absolutely clear.

Reply:
I've done the thing I see a lot of tech docs do where they portray a URL as "/your/url/path/". Whatcha reckon?

Reply:
Looks good!